### ISSUE TITLE
Opening internal Eclipse browser Feels archaic and doesn't buy developer much

### ISSUE THREAD

**Issue by branflake2267** [GoogleCloudPlatform/google-cloud-eclipse#940]
I think clicking on the console urls and opening an internal Eclipse browser doesn't by the target audience developer much at all. That said, opening the internal browser by default after server starts as well has the same effect. I think it feels like a digression in development because it doesn't have access to development tools and adds an unneeded step to getting debugging going. 

That said I do have a bias with the GWT Eclipse Plugin so I could have a prejudice here.  Feel free to close this, I won't mind, I'm expression an opinion on workflow from my angle. 

The target audience, web developers, I think will be expecting to be debugging in their favorite browser, the default browser. And opening in Eclipse sort of feels like an insult having no tools for dev tools for development. I think the only effective links to the internal web browser would be the local app engine dashboard. 

Example of the console click opens the internal Eclipse browser that shows web page. 
![screen shot 2016-11-03 at 8 22 11 am](https://cloud.githubusercontent.com/assets/1326504/19972398/25b445e4-a19f-11e6-9d0a-9b00d37a4726.png)

**Comment by branflake2267** [comment]
Preview of the server runtime workflow with GWT app: https://www.youtube.com/watch?v=r2vCjmmAMmY&list=PLBbgqtDgdc_TqzA-qXrjgTFMC_6DKAQyT&index=8&ab_channel=BrandonDonnelson

**Comment by briandealwis** [comment]
We respect the user's default web browser settings.  These can be changed with _Preferences > General > Web Browser_ and selecting _Use external web browser_.

JBoss developers seem to use the internal browser extensively.  I like using it too, for quick views.  

It does look like we can have our `BrowserSupportBasedHyperlink` implement `IHyperlink2` and get the SWT `Event` that triggered the activation.  We could check the modifier state and allow a control-click to open the external browser, and similarly an shift-click to open the internal browser.

**Comment by briandealwis** [comment]
Looks like we could also [respond to a double-click](https://github.com/excelsior-oss/xds-ide/blob/64f1c09b5a2e1ae173d199b23ef55f7193e7bf2b/product/com.excelsior.xds.ui/src/com/excelsior/xds/ui/console/XdsConsole.java#L299).

**Comment by akerekes** [comment]
I like the idea of examining the modifier (and also possibly considering middle click as request for external browser).

**Comment by branflake2267** [comment]
Right click could be an option. This has been done in the GWT Dev 

![screen shot 2016-11-03 at 9 22 59 am](https://cloud.githubusercontent.com/assets/1326504/19974875/355e681e-a1a7-11e6-998c-97f50bca02f1.png)
Mode View.

**Comment by briandealwis** [comment]
Should be able to open a [context menu programmatically](http://www.java2s.com/Code/Java/SWT-JFace-Eclipse/Showapopupmenuwaitforittoclose.htm) from the `IHyperlink2`-provided `Event`.

**Comment by elharo** [comment]
Right-click I believe. Double-click/Middle-click is just confusing since there's no reason for the user to expect this. That said, let's take this as feedback and see what other users think. If lots of folks mention he same thing, we'll implement it. Otherwise, probably not.
